### DISCUSSION TITLE
Error 500 when installing single project msix package extension for vs 2022

### DISCUSSION THREAD

**Discussion by BreyerW** [microsoft/WindowsAppSDK#1116] [General] [answered]
Title. Since error is just 500 (internal error) i dont know what else to add. Im using win10 and vs 2022. Oh and im trying to install it for maui winui3.

**Comment by andrewleader** [comment, marked as answer]
Hi @BreyerW, sorry we missed this question! Can you try the single project MSIX package again using the latest version of VS 2022? It's working in our recent tests, but might have not been working 27 days ago!

**Reply by BreyerW** [reply]
Yes it works with latest preview
